gitter.py

```diff
@@ -239,6 +239,8 @@ def __init__(self, config):
                              'Accept': 'application/json'}
         self.bot_identifier = self._get_bot_identifier()
 
+        self.joined_rooms = joined_rooms
```

Comment:
Lets make this private, since things outside the class should never mess with it (so `self._joined_rooms` instead).